src/indev/lv_indev_private.h

```diff
@@ -112,6 +112,9 @@ struct _lv_indev_t {
                                       here by the buttons*/
     lv_event_list_t event_list;
     lv_anim_t * scroll_throw_anim;
+
+    lv_indev_gesture_type_t gesture_type;
+    void * gesture_data;
```

Comment:
Probably this could be `lv_indev_gesture_recognizer_t *` since that's the only thing assigned to it but it's ok.